src/contracts/CreateUserParams.ts

```diff
@@ -0,0 +1,51 @@
+import { IsNotEmpty, IsOptional, IsString, Length } from "class-validator";
+import { BaseValidate } from "../shared/BaseValidate";
+
+type CreateUserParamsProps = {
+  name: string;
+  email: string;
+  cpf: string;
+  password: string;
+  avatar?: string;
+};
+
+export class CreateUserParams extends BaseValidate {
+  @Length(1, 255, { message: "A propriedade 'name' deve ter entre 1 e 255 caracteres" })
```

Comment:
CreateUserParams? não deveria ser algo como CreateUser ou se isso for montar e validar os campos para criar um User, deveria ser InputCreateUser ao meu ver -> Input (entrada) Create (para criar) User (usuário) -> entrada para criar um usuário

Talvez o `Create` possa ser removido por ele ser o unico método a ser usado dessa classe, logo na hora de usar vai ser algo como -> `InputUser.create(...params)` e não `CreateUserParams.create(...params)`